api/oas_schemas.py

```diff
@@ -0,0 +1,874 @@
+import textwrap
+
+from decouple import config
+from drf_spectacular.utils import OpenApiExample, OpenApiParameter
+
+from api.serializers import (
+    ListOrderSerializer,
+    OrderDetailSerializer,
+    StealthSerializer,
+)
+
+EXP_MAKER_BOND_INVOICE = int(config("EXP_MAKER_BOND_INVOICE"))
+RETRY_TIME = int(config("RETRY_TIME"))
+PUBLIC_DURATION = 60 * 60 * int(config("DEFAULT_PUBLIC_ORDER_DURATION")) - 1
+ESCROW_DURATION = 60 * int(config("INVOICE_AND_ESCROW_DURATION"))
+BOND_SIZE = int(config("DEFAULT_BOND_SIZE"))
+
+
+class MakerViewSchema:
```

Comment:
There is so much work and attention to detail here that is overwhelming. I say it is accurate to over 99% and the few things that are not fully accurate are not worth fixing. All in all I would suggest to merge as is: having this doc for the v0 API is a huge step forward.

Reply:
hehe.. Appreciate you noticing :grin: . Tried my best to me accurate and detailed as possible.